mintpy/defaults/smallbaselineApp.cfg

```diff
@@ -62,6 +62,7 @@ mintpy.load.bperpFile      = auto  #[path pattern of 2D perpendicular baseline f
 ## multilook while loading data with nearest interpolation, to reduce dataset size
 mintpy.load.ystep          = auto    #[int >= 1], auto for 1 - no multilooking
 mintpy.load.xstep          = auto    #[int >= 1], auto for 1 - no multilooking
+mintpy.load.stepMethod     = auto    #[nearest, mean, median], auto for nearest - lines/rows skipping approach
```

Comment:
How about using the following as we discussed before? `mintpy.load.stepMethod` does not sound very intuitive to me.

```
mintpy.multilook.ystep
mintpy.multilook.xstep
mintpy.multilook.method
```

Reply:
Sounds good. I will do it now.

Reply:
This is done now.